src/ngraph_skip_assert.cc

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#include "ngraph_skip_assert.h"
+
+#include "tensorflow/core/graph/graph.h"
+#include "tensorflow/core/graph/node_builder.h"
+#include "tensorflow/core/graph/types.h"
+
+#include "ngraph_utils.h"
+
+using namespace std;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+//
+// Main entry point for skip assert.
+//
+Status SkipAssert(Graph* graph) {
+  for (auto node : graph->op_nodes()) {
+    if (node->type_string() == "Assert") {
+      NGRAPH_VLOG(4) << "Checking: " << node->name();
+      for (auto edge : node->out_edges()) {
+        if (edge->IsControlEdge()) {
+          NGRAPH_VLOG(4) << "Control edge: " << node->name();
+          graph->RemoveControlEdge(edge);
```

Comment:
We have already printed `node->name()` in an earlier line.
`edge->DebugString()` might be a more informative print. 

Also we could remove `NGRAPH_VLOG(4) << "Control edge removed. ";` since `RemoveControlEdge` has no chance of failure.

Reply:
Yes, we can change the log in line 39, to "Removing control edge" << edge->DebugString()